packages/blitz/src/renderer/render.rs

```diff
@@ -1091,4 +1093,61 @@ impl ElementCx<'_> {
     ) {
         unimplemented!()
     }
+
+    fn draw_input(&self, scene: &mut Scene) {
+        if self.element.local_name() == "input"
+            && matches!(self.element.attr(local_name!("type")), Some("checkbox"))
+        {
+            let checked: bool = self
+                .element
+                .attr(local_name!("checked"))
+                .and_then(|c| c.parse().ok())
+                .unwrap_or_default();
+
+            // TODO this should be coming from css accent-color, but I couldn't find how to retrieve it
```

Comment:
`accent-color` can be enabled here: https://github.com/servo/stylo/blob/main/style/properties/longhands/inherited_ui.mako.rs#L94 by changing `engines="gecko"` to `engines="gecko servo"`.

Such a change could be submitted to the DioxusLabs fork of Stylo. We are currently using the [`enable-table-moz-center-style-adjust`](https://github.com/DioxusLabs/stylo/tree/enable-table-moz-center-style-adjust) branch. So changes should be based on that branch (we should probably be more principled about using a `blitz` branch or similar).